packages/block-directory/src/store/reducer.js

```diff
@@ -33,6 +32,24 @@ export const downloadableBlocks = ( state = {
 				hasPermission: true,
 				isRequestingDownloadableBlocks: false,
 			};
+	}
+	return state;
+};
+
+/**
+ * Reducer managing the installation and deletion of blocks.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export const blockManagement = ( state = {
+	hasPermission: true,
+	installedBlockTypes: [],
+	items: [],
```

Comment:
Not introduced by this PR, but I haven't looked at this code for a while, and it's now not clear to me what `items` represents and how it differs from `results` and `installedBlockTypes`.

Some extra documentation or an improvement to the naming are two possible solutions.

Reply:
Again, I should've read the PR description 😄 

It might be worth checking the original PR to see if it was used. I know @gziolo removed some block directory code in https://github.com/WordPress/gutenberg/pull/17576, which might have been where it was used.

Reply:
It doesn't look like I modified stores in the referenced PR.